postgres/tests/test_metadata.py

```diff
@@ -117,6 +117,8 @@ def test_collect_schemas(integration_check, dbm_instance, aggregator, use_defaul
     schemas_want = {
         'public',
         'public2',
+        'datadog',
+        'hstore',
```

Comment:
I think we might want logic here to skip `hstore` schema if this is an empty schema from the extension

Reply:
oh...nvm I see that in our testing harness we're creating an empty schema called `hstore` to test extension loading of `hstore` from another schema. It's quite confusing that we named this schema after the extension 😅